### DISCUSSION TITLE
Backup of historical data in Energy Dashboard

### DISCUSSION THREAD

**Discussion by fico9999** [lavinir/hassio-onedrive-backup#280] [General]
Hi everyone,

Maybe this is a rookie question, but how can I make sure that the historical data in my Energy Dashboard is backed up properly using Home Assistant OneDrive Backup? Is this included in the OneDrive backups by default, or do I need to do something extra?

Actually, this is the only thing that I am worried about. All the rest is less important for me.

Thank you in advance!

**Comment by lavinir** [comment]
This actually has nothing to do with the OneDrive backup addon itself, as it is not creating backups in a proprietary way. The backups are your regular Home Assistant backups. Regarding whether energy historical data is included and in what circumstances I'm not sure but found this in the forums, maybe it can help you:
https://community.home-assistant.io/t/backup-option-for-energy-dashboard/355533/5
